controllers/front/payment.php

```diff
@@ -32,7 +32,7 @@ public function paymentAction($payMethod) {
 			'return_url' => $redirectUrl,
 			'cancel_url' => $cancelUrl,
 			'postback_url' => $callbackUrl,
-			'description' => strval($cart->id),
+			'description' => 'Order: ' . strval($cart->id),
```

Comment:
Nice one!

I found in Magento we did with `'description' => $order->getRealOrderId() . ' - ' . $order->getStore()->getFrontendName()`

Reply:
we do not have OrderId on Presta. what is getFrontendName?

Reply:
it is the shop name

Reply:
we also included the shop name for Prestashop before, but it raised the error: description is longer than 35. here: https://github.com/paypronl/prestashop-payments-plugin/pull/8/files
Interesting how Magento does not raise it.

Reply:
hmm, yes, we should be careful, probably no merchants complain for it for now. I'll write it down for now, later we can find some time to check and fix it for all plugins.